dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -166,6 +166,18 @@ const CommentFooter = ({
 	);
 };
 
+const getImage = ({
+	imageUrl,
+	avatarUrl,
+}: {
+	imageUrl?: string;
+	avatarUrl?: string;
+}): { type: CardImageType; src: string } | undefined => {
+	if (avatarUrl) return { type: 'avatar', src: avatarUrl };
```

Comment:
This logic looks to be wrong? The previous lines:
```ts
if (imageUrl && avatarUrl) {
	imageType = 'avatar';
}
```

So this should be 
```ts
if (avatarUrl && imageUrl) return { type: 'avatar', src: avatarUrl };
```

That said I don't think I fully understand _why_ this logic is the way it is

Reply:
I could not figure it out either, and this causes no visual difference, so tempted to simplify it like so.

If we want to capture a missing `imageUrl` and a set `avatarUrl`, we should have a story against it.

Reply:
I agree the previous logic is a bit weird and I prefer the new version. I'll leave this up to both of you. On my side, I'm happy with this PR so I have approved.